nx/test/nx/lin_alg_test.exs

```diff
@@ -159,10 +159,36 @@ defmodule Nx.LinAlgTest do
       assert_all_close(Nx.dot(q, r), t, atol: 1.0e-10)
     end
 
+    test "works with complex matrix" do
+      import Nx, only: :sigils
+
+      t = ~M[
+        1 0 1i
+        0 2 -1i
+        1 1 1
+      ]
+
+      {q, r} = Nx.LinAlg.qr(t)
+
+      assert_all_close(q, ~M[
+        -0.7070 0.2357  0.6666
+         0      -0.9428   0.3333
+        -0.7070 -0.2357  -0.6666
```

Comment:
During tests, I needed to copy and paste complex tensors from the terminal but couldn't just paste them into the sigil due to the brackets for each row.

Perhaps we can allow for that through a regex? Something like:

String.replace(string, ~r/\[([^\]+)\]/, "\\1")

Reply:
Another option is to print the tensor differently. Using a similar format as ~M. Higher dimensional tensors are printed as "lists" of matrices.

Reply:
So we would print all tensors like this every time?

Reply:
I don't know, somethjing to discuss :)